packages/features/portfolio/src/side-drawers/convert-cash-to-n-tokens/error-msgs.tsx

```diff
@@ -0,0 +1,12 @@
+import { defineMessages } from 'react-intl';
+
+export const errorMsgs = defineMessages({
+  insufficientBalance: {
+    defaultMessage: 'balance is insufficient',
+    description: 'error message',
+  },
+  inputGreaterThanZero: {
```

Comment:
I do not believe that the CurrencyInput box allows you to even input negative numbers so this error message is unnecessary